3.9/Dockerfile

```diff
@@ -0,0 +1,132 @@
+FROM metabrainz/consul-template-base:v0.18.5-2
+
+# This Dockerfile is based on
+# https://github.com/docker-library/python/blob/393ba9b3/3.9/buster/Dockerfile
+
+# Ensure that local Python build is preferred over whatever might come with the base image
+ENV PATH /usr/local/bin:$PATH
+
+# http://bugs.python.org/issue19846
+# > At the moment, setting "LANG=C" on a Linux system *fundamentally breaks Python 3*, and that's not OK.
+ENV LANG C.UTF-8
+
+# Runtime dependencies. This includes the core packages for all of the buildDeps listed
+# below. We explicitly install them so that when we `remove --auto-remove` the dev packages,
+# these packages stay installed.
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+                       ca-certificates \
+                       netbase \
+                       libbz2-1.0 \
+                       libexpat1 \
+                       libffi6 \
+                       libgdbm3 \
+                       liblzma5 \
+                       libncursesw5 \
+                       libreadline6 \
+                       libsqlite3-0 \
+                       libssl1.0.0 \
+                       libuuid1 \
+                       tcl \
+                       tk \
+                       zlib1g \
+	&& rm -rf /var/lib/apt/lists/*
+
+ENV GPG_KEY E3FF2839C048B25C084DEBE9B26995E310250568
+ENV PYTHON_VERSION 3.9.1
+
+# The list of build dependencies comes from the python-docker slim version:
+# https://github.com/docker-library/python/blob/32b69d61f6/3.8/buster/slim/Dockerfile#L29
```

Comment:
```suggestion
# https://github.com/docker-library/python/blob/b5db377b08/3.9/buster/slim/Dockerfile#L33
```

The only difference I see is the [add of `libbluetooth-dev`](https://github.com/docker-library/python/blob/b5db377b082be21f3f799e84bbec73abe2ec9ad0/3.9/buster/slim/Dockerfile#L35). Not sure how relevant this is for us.

Reply:
ah, I missed this url, thanks. I think we can skip libbluetooth in this case because we won't need it on the servers